components/src/core/Drawer/DrawerNavItem/DrawerNavItem.tsx

```diff
@@ -179,11 +180,14 @@ const ActiveComponent = styled(Box, {
 
 const NestedListGroup = styled(List, {
     shouldForwardProp: (prop) => prop !== 'nestedBackgroundColor',
-})<Pick<DrawerNavItemProps, 'nestedBackgroundColor'>>(({ nestedBackgroundColor, theme }) => ({
-    backgroundColor: nestedBackgroundColor || (theme.palette.mode === 'light' ? white[200] : darkBlack[500]),
-    paddingBottom: 0,
-    paddingTop: 0,
-}));
+})<Pick<DrawerNavItemProps, 'nestedBackgroundColor'>>(({ nestedBackgroundColor }) => {
+    const colorScheme = useColorScheme();
```

Comment:
This is unfortunate...if we rely on using `useColorScheme` then we aren't going to be able to support v5 of MUI. Can we find a way to use theme.applyStyles instead of relying on useColorScheme to get the mode?